Chirp/Pages/MorePage/MorePageViewController.swift

```diff
@@ -0,0 +1,115 @@
+    let userProfileImage: UIImageView = {
+        let imageView = UIImageView()
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        imageView.image = UIImage(named: "mountains")
+        return imageView
+    }()
+    
+    let userNameLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.text = "Cameron Elizabeth Williamson"
+        label.numberOfLines = 2
+        label.font = UIFont.systemFont(ofSize: 22, weight: .bold)
+        return label
+    }()
+    
+    let userPhoneLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.text = "+91 999 2222 433"
+        label.font = UIFont.systemFont(ofSize: 18, weight: .thin)
+        return label
+    }()
+    
+    let moreTableView: UITableView = {
+        let tableView = UITableView()
+        tableView.translatesAutoresizingMaskIntoConstraints = false
+        return tableView
+    }()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        title = "More"
+        doBasicSettings()
+    }
+
+    private func doBasicSettings() {
+        view.backgroundColor = .white
+        addSubviews()
+        setupTableView()
+        setupConstraints()
+        setupUserImage()
+    }
+    
+    private func addSubviews() {
+        view.addSubview(userProfileImage)
+        view.addSubview(userNameLabel)
+        view.addSubview(userPhoneLabel)
+        view.addSubview(moreTableView)
+    }
+    
+    private func setupTableView() {
+        moreTableView.delegate = self
+        moreTableView.dataSource = self
+        moreTableView.register(MorePageTableViewCell.self, forCellReuseIdentifier: MorePageTableViewCell.identifier)
+        moreTableView.rowHeight = 80
+    }
+
+    private func setupConstraints() {
+        NSLayoutConstraint.activate([
+            userProfileImage.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor, constant: 125),
+            userProfileImage.leadingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leadingAnchor, constant: 25),
+            userProfileImage.widthAnchor.constraint(equalToConstant: 110),
+            userProfileImage.heightAnchor.constraint(equalToConstant: 110),
+            
+            userNameLabel.topAnchor.constraint(equalTo: userProfileImage.topAnchor, constant: 15),
+            userNameLabel.leadingAnchor.constraint(equalTo: userProfileImage.trailingAnchor, constant: 20),
+            userNameLabel.trailingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.trailingAnchor, constant: -25),
+            
+            userPhoneLabel.bottomAnchor.constraint(equalTo: userNameLabel.bottomAnchor, constant: 30),
+            userPhoneLabel.leadingAnchor.constraint(equalTo: userProfileImage.trailingAnchor, constant: 20),
+            userPhoneLabel.trailingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.trailingAnchor, constant: -25),
```

Comment:
Can we instead create this as a separate UIView and then add the view into the tableView header so that it scrolls with the content?